testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -43,3 +43,28 @@ def test_transform_StoR_fail():
 
     with pytest.raises(TypeError, match='S must be of type float32'):
         r = mda.lib.distances.transform_StoR(s, box)
+
+def test_capped_distance_withpbc():
+	box = np.array([1., 1., 1., 90., 90., 90.], dtype=np.float32)
+	point1 = np.array([0.1, 0.1, 0.1], dtype=np.float32)
```

Comment:
these tests aren't suitable.  the coordinates are nearly always going to be arrays of coords (ie shape `(n, 3)`) not single points.  If you need to, generate ~100 random coords with a fixed seed, do it the slow way with `distance_array` then check that this new method agrees.  Use a seed which ensures that the pbc option makes a difference and the results have meaningful size (ie not always 0/all)